neclib/devices/ad_converter/cpz3177.py

```diff
@@ -7,6 +7,57 @@
 
 
 class CPZ3177(ADConverter):
+
+    """a/d converter, which can convert by 32 or 64 channels.
+
+    Notes
+    -----
+    Configuration items for this device:
+
+    rsw_id : {0, 1, ..., 16} or {"0", "1", ..., "9", "A", ..., "F"}
+        Board identifier. This should be set to the same value as the rotary switch
+        "RSW1" mounted on the side of the board. The board is shipped with default RSW1
+        setting of 0. This ID would be non-zero, when multiple PCI board of same model
+        are mounted on a single FA (Factory Automation) controller.
+
+    ave_num : int
+        Sampling data number to calculate average of data. This source code not only
+        measure the actual voltage, but also calculate the average of data
+        in order to measure a faint voltage. It should not be define too large number
+        against sampling frequency.
+
+    smpl_freq : int
+        Sampling frequency. Number of measuring data per 1 second.
+        It should not be too small number against number of data for
+        calculating average.
+
+    single_diff : {"SINGLE" or "DIFF"}
+        Input type of voltage. "SINGLE" is Single-ended input, "DIFF" is
+        "Differential input".
+
+    all_ch_num : int
+        Number of channel using for measuring voltage. This number must be  the same
+        as the number of setting channel in smpl_ch_req. The maximum number is
+        32 in differential input, 64 in single-ended input. Please read the manual
+        of this board for wiring design.
+
+    smpl_ch_req : List[Dict[str, int], Dict[str, int]...]
+        List of measuring range. The number of setting channel must be the same as
+        all_ch_num. In addition, it must be defined ch1 through the maximum used
+        channel number. For example, channel number: 3, 5, 9, 12 are used, then
+        smpl_ch_req must be set ch1 through ch12.
```

Comment:
```suggestion
    smpl_ch_req : List[Dict[str, Union[int, str]]]
        List of measurement range, in format
        ``{ ch_no = int(channel_number), range = str(range_specifier) }``. The length of
        the list must be the same as ``all_ch_num``. In addition, it must be defined
        from ch1 through the maximum channel number to be used. For example, when
        channels [3, 5, 9, 12] are used, ``smpl_ch_req`` must be set from ch1 through
        ch12.
```

`range_specifier` について、受け付ける値のリストを追加するとより親切なドキュメントになると思います。

Reply:
完全に失念していました。PCIボードに物理的に実装されているスイッチとの対応も追記しましたので確認をお願いします。